source/system/String.ooc

```diff
@@ -26,6 +26,7 @@ String: class {
 		this _buffer free()
 		super()
 	}
+	safeFree: func { if (!this _isLiteral) this free() } // Will not free literals automatically
```

Comment:
If someone gave me a string to keep, I would like to free only if it is not reused.